bin/release.sh

```diff
@@ -31,8 +31,10 @@ $BIN_DIR/simple-test.sh $TAG
 
 set -x
 docker tag kimai/kimai2:fpm-alpine-${TAG}-prod kimai/kimai2:latest
```

Comment:
One question, outside that PR - but I stumbled upon it while reading through the docs right now... 
The doc at https://tobybatch.github.io/kimai2/dev-instance.html#dev-instances say that `kimai/kimai2:latest` is a apache-dev instance. 
Are the docs wrong here?